pkg/router/listen_path_parameter_name_extractor.go

```diff
@@ -0,0 +1,30 @@
+package router
+
+import "regexp"
+
+const (
+	parameterMatchRule = `\{([a-z\d]+?)\}`
```

Comment:
How about `/\{[^/}]+\}/`?

Reply:
I need the group to extract the parameter inside the branches

Reply:
The group is in mine as well :)

Reply:
Now it is, I saw you 😛